jakartaee/faces
# Signature file and sigtest plugin update

To generate the signature file : 
```
cd tck/faces-signaturetest
mvn clean install -Precord-signature
```
The signature file will be generated at faces-signaturetest/target/

To run the signature test :
- Place the generated signature file at faces-signaturetest/src/test/resources/ee/jakarta/tck/faces/signaturetest/
- Update the sig-test.map file with version of the signature file

```
cd tck/faces-signaturetest
mvn clean verify 
```
